trunner/harness/psh.py

```diff
@@ -52,16 +52,25 @@ class ShellHarness(IntermediateHarness):
 
     """
 
-    def __init__(self, dut: Dut, prompt: str, cmd: Optional[List[str]] = None, prompt_timeout: Optional[int] = -1):
+    def __init__(
+        self,
+        dut: Dut,
+        prompt: str,
+        cmd: Optional[List[str]] = None,
+        prompt_timeout: Optional[int] = -1,
+        suppress_dmesg: Optional[bool] = True,
```

Comment:
Type is not `Optional`, only `bool` (takes True / False values, not True / False / None). 

The same with `prompt_timeout` I assume (not related to this PR, not sure when it was introduced)